addons/etcd/scripts/switchover-with-candidate.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -exo pipefail
+
+leader_endpoints=${KB_CONSENSUS_LEADER_POD_FQDN}:2379
+candidate_endpoints=${KB_SWITCHOVER_CANDIDATE_FQDN}:2379
+
+candidateID=$(etcdctl --endpoints=$candidate_endpoints endpoint status | awk -F', ' '{print $2}')
+etcdctl --endpoints=$leader_endpoints move-leader $candidateID
```

Comment:
why not use tls info like you do in role-probe.sh

Reply:
solved